Source/Impersonator/configuration/changes/change_notifier.go

```diff
@@ -0,0 +1,82 @@
+package changes
```

Comment:
This configuration stuff seems similar to what we're using in the backend. Maybe we should consider dragging it out into it's own module?

Reply:
That would be preferable yes - but it involves a bit of trickery or restructuring of the folder structure in this repository to make it work (there is no npm/nuget/.. for go). So I would postpone it until we get more stuff in Go and make a Go module publishing thing that we can use

Reply:
But it seems like it solves a more generic problem with viper configurations? We could pull it out to [https://github.com/dolittle-entropy](dolittle-entropy) and keep it separate. Properly name the go-module GitHub.com/dolittle-entroy/module-name. Or host it from our Dolittle.io site 😛

Reply:
Yes, I agree - we should host it on dolittle.io. But I'm just saying that I think we should wait with that change for later. It's not really a big maintenance problem for now since all of our go code is here 😄